requirements.txt

```diff
@@ -2,8 +2,8 @@
 v1pysdk-unofficial==0.4.post4
 
 helga>=1.6.0
-Twisted==13.1.0
-pymongo==2.5.2
-httplib2==0.9.2
-oauth2client==2.0.1
-expiringdict==1.1.3
+Twisted>=13.1.0
+pymongo>=2.5.2
+httplib2>=0.9.2
+oauth2client>=2.0.1
+expiringdict>=1.1.3
```

Comment:
Man Github makes it hard to actually find this comment.

Ok +1 on py26 removal then.

Deps should be unpinned in the setup.py, which is what will be used when you pip install the package. requirements.txt is for... umm.. running the test suite and "deployment" which doesn't exactly apply to this module.

So yeah, dep management is hard. But I always (try to) do unpinned in setup.py and pinned in requirements.txt